### DISCUSSION TITLE
I can't update the password of the connection through the API

### DISCUSSION THREAD

**Discussion by LinYuHeng-Con** [apache/airflow#25818] [Q&A] [answered]
![airflow](https://user-images.githubusercontent.com/48077349/185598712-3ac9520c-ede6-4d3c-b19e-25d65647d5c3.PNG)

I use the update connection information API, but I find that the connection password cannot be modified.

![a](https://user-images.githubusercontent.com/48077349/185598983-375b9bff-56b9-4beb-a64b-07a28c38fde9.PNG)

After modification, the password of the connection is still empty

**Comment by potiuk** [comment, marked as answer]
Password is - I believe - write-only from the UI and never sent as response (all of that for security reasons). Are you sure it is not changed? Did you check if it did not change?

**Reply by LinYuHeng-Con** [reply]
I'm sure my password hasn't been updated.I can only change my password through the UI

**Reply by LinYuHeng-Con** [reply]
sry,it's my problem. I found that the password has been updated, but it is not displayed on the UI interface
